test/definitions/features/installer_test.rb

```diff
@@ -11,6 +11,7 @@
     before do
       installer_config_dir(["#{data_dir}/installer/simple_config"])
       mock_installer_package('foreman-installer')
+      installer_args
```

Comment:
You can directly place line here than method.

```suggestion
     Features::Installer.any_instance.stubs(:installer_arguments).returns(' --disable-system-checks \
                                                                         --upgrade')
```